vaccine/vaccine_reg_ussd.py

```diff
@@ -275,7 +275,13 @@ async def state_confirm_profile(self):
         return MenuState(
             self,
             question="\n".join(
-                ["Confirm the following:", "", f"{first_name} {surname}", id_number]
+                [
+                    "Confirm the following:",
+                    "",
+                    f"{first_name} {surname}",
+                    id_number,
+                    "\n",
```

Comment:
I think this will add a double blank space (due to the `"\n".join`). We probably just want this to be `"",`